bats/tests/helpers/load.bash

```diff
@@ -84,6 +84,11 @@ call_local_function() {
 }
 
 setup_file() {
+    # We require bash 5; bash 3.2 (as shipped by macOS) seems to have
+    # compatibility issues.
+    if semver_gt 5.0.0 "$(semver "$BASH_VERSION")"; then
+        fail "Bash 5.0.0 is required; you have $BASH_VERSION"
```

Comment:
Technically 5.0.0 **or later** is required, but good enough.

Reply:
Yes; this triggers if `5.0.0 > $BASH_VERSION`, which means we succeed if `$BASH_VERSION >= 5.0.0`.

Reply:
Yeah, my comment was that the error message requires an exact version match whereas the implementation just checks for a minimum version. Since the implementation is correct, and nobody will likely ever see the error message, we should be good.